tests/authentication.py

```diff
@@ -16,51 +13,41 @@
 class TestTokenEndpoint(unittest.TestCase):
     @classmethod
     def setUpClass(cls) -> None:
-        config = Config()
+        cls.config = Config()
         cls.cm = CrudMethods()
-        cls.url = config.URL
-        cls.username = config.USR
-        cls.password = config.PWD
 
     def test_01_get_api_key_response_200(self):
-        r = self.__class__().cm.request_method(
-            "POST",
-            self.__class__().url,
+        r = self.__class__().cm.create(
+            self.config.URL,
             "/token",
             "",
             {HttpHeaders.ACCEPT: "application/json"},
-            HTTPBasicAuth(self.username, self.password),
+            HTTPBasicAuth(self.config.USR, self.config.PWD),
         )
         with soft_assertions():
             assert_that(r.status_code).is_equal_to(200)
```

Comment:
skoro juz masz pytesta, to warto byłoby skorzystać z niego do odpalania testów, i wtedy tutaj można po prostu zrobić `assert r.status_code == 200` i pytest, w przypadku faila, to odpowiednio ładnie opisze.

Reply:
with soft_assertion() uzywam jesli chce sprawdzic nie tylko status code, bo możesz sprawdzić wiele asserci na raz czyli np. jeśli dev źle zakoduje i zwróci zły kod błędu, to nie sprawdzisz czy przyszły dane)